packages/components/src/Button/button.stories.tsx

```diff
@@ -69,17 +68,21 @@ const gridParameters = {
   },
 };
 
-export const AllVariants = () => getAllVariantsWithStates("button", "Button");
-AllVariants.parameters = gridParameters;
+export const AllVariants = {
```

Comment:
Generally I think we should explicitly type stories - i.e.

```ts
export const AllVariants: StoryObj<Args> = {
```

But for these stories that don't use or set args, it doesn't really matter.

Reply:
Could you explain why we should explicitly type every story?

Reply:
Mostly so it forces us to get the `args` right (if we are using any).

Also I think `composesStories` requires them to be annotated to infer types in tests.

Reply:
Maybe we should think about a lint rule for that 🤔